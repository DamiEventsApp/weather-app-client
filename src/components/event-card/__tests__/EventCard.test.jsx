import React from 'react';
import { shallow } from 'enzyme';
import EventCard from '../EventCard.component.jsx';
import '../../../../setupTests.js';

describe('EventCard', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<EventCard />);
        expect(wrapper).toMatchSnapshot();
    });
});