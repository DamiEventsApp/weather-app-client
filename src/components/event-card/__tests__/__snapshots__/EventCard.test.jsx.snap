// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventCard renders correctly 1`] = `ShallowWrapper {}`;
